docs/user/summary.md

```diff
@@ -0,0 +1,1115 @@
+tenancy.contactrole: Nautobot Model Not Found
+tenancy.contact: Nautobot Model Not Found
+tenancy.contactassignment: Nautobot Model Not Found
+dcim.inventoryitemrole: Nautobot Model Not Found
+dcim.inventoryitemtemplate: Nautobot Model Not Found
+dcim.cabletermination: Nautobot Model Not Found
+dcim.virtualdevicecontext: Nautobot Model Not Found
+ipam.fhrpgroup: Nautobot Model Not Found
+ipam.fhrpgroupassignment: Nautobot Model Not Found
+ipam.servicetemplate: Nautobot Model Not Found
+ipam.l2vpn: Nautobot Model Not Found
+ipam.l2vpntermination: Nautobot Model Not Found
+extras.report: Nautobot Model Not Found
+extras.script: Nautobot Model Not Found
+extras.journalentry: Nautobot Model Not Found
+extras.configrevision: Nautobot Model Not Found
+extras.savedfilter: Nautobot Model Not Found
+extras.cachedvalue: Nautobot Model Not Found
+extras.branch: Nautobot Model Not Found
+extras.stagedchange: Nautobot Model Not Found
+users.adminuser: Nautobot Model Not Found
+wireless.wirelesslangroup: Nautobot Model Not Found
+wireless.wirelesslan: Nautobot Model Not Found
+wireless.wirelesslink: Nautobot Model Not Found
+django_rq.queue: Nautobot Model Not Found
+secrets.secret: Nautobot Model Not Found
+secrets.secretrole: Nautobot Model Not Found
+secrets.userkey: Nautobot Model Not Found
+secrets.sessionkey: Nautobot Model Not Found
+```
+
+## Content Types Back Mapping
+
+This section shows back mapping deviations from Nautobot content types to NetBox content types. Only those content types that differ between NetBox and Nautobot are shown.
+
+"Ambiguous" means that there are multiple NetBox content types that map to the same Nautobot content type.
+
+```
+- Content Types Back Mapping: --------------------------------------------------
+  Back mapping deviations from Nautobot content type to the source content type
+extras.role => Ambiguous
+users.user => auth.user
+dcim.locationtype => Ambiguous
+dcim.location => Ambiguous
+ipam.prefix => Ambiguous
+```
+
+## Fields Mapping
+
+The last section shows fields mapping from NetBox to Nautobot, grouped by NetBox content type.
+
+Each line consists of the following:
+
+- NetBox field name.
+- `(DATA)` marks fields that are present in the NetBox data.
+- `(CUSTOM)` marks fields that have a custom import logic defined instead of the default one.
+- `=>` as a separator.
+- Nautobot field name.
+- Field type in parentheses with the following special cases:
+    - `SKIPPED` means the field is intentionally skipped by the importer.
+    - `(ReadOnlyProperty')` marks fields that are read-only properties in Nautobot and can't be imported.
+    - `(PrivateProperty)` marks fields that are prefixed with an underscore `_` and are considered private properties. Those fields are not imported.
+    - `(NotFound)` indicates the field is not found in Nautobot and can't be imported.
+    - `(DoNotImportLastUpdated)` are fields that are not imported as they are automatically updated by Nautobot.
```

Comment:
the word seems only for "LastUpdated" not for any field updated by Nautobot

Reply:
Updated the documentation to address this.